### DISCUSSION TITLE
Add API ability to toggle releases/packages/environments from homepage of repo

### DISCUSSION THREAD

**Discussion by vincerubinetti** [community/community#39800] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

The rest API allows you to set many things about a repo:

https://docs.github.com/en/rest/repos/repos#update-a-repository

Including repo homepage stuff like `homepage` and `description`, and repo settings like `has_wiki`.

But I cannot find any way to programmatically set the values of these checkboxes:

<img width="158" alt="Screenshot 2022-11-22 at 8 49 45 PM" src="https://user-images.githubusercontent.com/8326331/203454906-60620bf5-cd88-415d-9c27-304a890f3b56.png">

It's not a big deal, but these checkboxes are on by default, and it adds a lot of visual cruft to the homepage that is very often not needed. For example, I am creating a public website template repo, which will never need any releases/packages/environments, and I'd like to shut these things off for the user (automatically when they fork the template, using GitHub actions) so they see less stuff in the UI and are less confused/distracted (the template is made for non-technical users).

**Comment by bestia-dev** [comment]
I would also like to modify these setting using the API and didn't find how to do it.
![image](https://github.com/community/community/assets/31509965/17fa7acc-2bb2-4281-92d7-35a7673fa7c0)
